gmm/gmm.dot

```diff
@@ -0,0 +1,27 @@
+digraph FSM {
```

Comment:
Can you please explain what this file does?

Reply:
The file is removed.

Reply:
The change is tested and docker build completes successfully.

```
$ make docker-build
for target in builder amf; do \
	DOCKER_BUILDKIT=1 docker build   \
		--target $target \
		--tag 5gc-$target:0.0.1-dev \
		--build-arg org_label_schema_version="0.0.1-dev" \
		--build-arg org_label_schema_vcs_url="https://github.com/gatici/amf.git" \
		--build-arg org_label_schema_vcs_ref="unknown" \
		--build-arg org_label_schema_build_date="2023-11-14T08:27:28Z" \
		--build-arg org_opencord_vcs_commit_date="unknown" \
		. \
		|| exit 1; \
done
[+] Building 32.2s (15/15) FINISHED                                                                                                                                                    
 => [internal] load build definition from Dockerfile                                                                                                                              0.1s
 => => transferring dockerfile: 38B                                                                                                                                               0.0s
 => [internal] load .dockerignore                                                                                                                                                 0.0s
 => => transferring context: 2B                                                                                                                                                   0.0s
 => [internal] load metadata for docker.io/library/golang:1.21.3-bookworm                                                                                                         0.6s
 => [internal] load build context                                                                                                                                                 0.7s
 => => transferring context: 33.63MB                                                                                                                                              0.7s
 => [builder  1/10] FROM docker.io/library/golang:1.21.3-bookworm@sha256:5bafbbb109f02aaf6b41ddc19f54919773c3006f1cbda1599112603367642f0e                                         0.0s
 => CACHED [builder  2/10] RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list                                                                   0.0s
 => CACHED [builder  3/10] RUN apt-get update                                                                                                                                     0.0s
 => CACHED [builder  4/10] RUN apt-get -y install apt-transport-https ca-certificates                                                                                             0.0s
 => CACHED [builder  5/10] RUN apt-get update                                                                                                                                     0.0s
 => CACHED [builder  6/10] RUN apt-get -y install gcc cmake autoconf libtool pkg-config libmnl-dev libyaml-dev                                                                    0.0s
 => CACHED [builder  7/10] RUN apt-get clean                                                                                                                                      0.0s
 => CACHED [builder  8/10] RUN cd /go/src && mkdir -p amf                                                                                                                         0.0s
 => [builder  9/10] COPY . /go/src/amf                                                                                                                                            1.2s
 => [builder 10/10] RUN cd /go/src/amf     && make all                                                                                                                           26.3s
 => exporting to image                                                                                                                                                            3.2s
 => => exporting layers                                                                                                                                                           3.1s
 => => writing image sha256:e20e1ce65be9b38152225c57a2535ada4a5456eed714d2058e18c366090a01aa                                                                                      0.0s 
 => => naming to docker.io/library/5gc-builder:0.0.1-dev                                                                                                                          0.0s
[+] Building 1.9s (23/23) FINISHED                                                                                                                                                     
 => [internal] load build definition from Dockerfile                                                                                                                              0.0s
 => => transferring dockerfile: 38B                                                                                                                                               0.0s
 => [internal] load .dockerignore                                                                                                                                                 0.0s
 => => transferring context: 2B                                                                                                                                                   0.0s
 => [internal] load metadata for docker.io/library/alpine:3.17                                                                                                                    0.4s
 => [internal] load metadata for docker.io/library/golang:1.21.3-bookworm                                                                                                         0.4s
 => [internal] load build context                                                                                                                                                 0.2s
 => => transferring context: 398.58kB                                                                                                                                             0.1s
 => [builder  1/10] FROM docker.io/library/golang:1.21.3-bookworm@sha256:5bafbbb109f02aaf6b41ddc19f54919773c3006f1cbda1599112603367642f0e                                         0.0s
 => [amf 1/7] FROM docker.io/library/alpine:3.17@sha256:f71a5f071694a785e064f05fed657bf8277f1b2113a8ed70c90ad486d6ee54dc                                                          0.0s
 => CACHED [builder  2/10] RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list                                                                   0.0s
 => CACHED [builder  3/10] RUN apt-get update                                                                                                                                     0.0s
 => CACHED [builder  4/10] RUN apt-get -y install apt-transport-https ca-certificates                                                                                             0.0s
 => CACHED [builder  5/10] RUN apt-get update                                                                                                                                     0.0s
 => CACHED [builder  6/10] RUN apt-get -y install gcc cmake autoconf libtool pkg-config libmnl-dev libyaml-dev                                                                    0.0s
 => CACHED [builder  7/10] RUN apt-get clean                                                                                                                                      0.0s
 => CACHED [builder  8/10] RUN cd /go/src && mkdir -p amf                                                                                                                         0.0s
 => CACHED [builder  9/10] COPY . /go/src/amf                                                                                                                                     0.0s
 => CACHED [builder 10/10] RUN cd /go/src/amf     && make all                                                                                                                     0.0s
 => CACHED [amf 2/7] RUN apk update                                                                                                                                               0.0s
 => CACHED [amf 3/7] RUN apk add -U vim strace net-tools curl netcat-openbsd bind-tools bash                                                                                      0.0s
 => CACHED [amf 4/7] WORKDIR /free5gc                                                                                                                                             0.0s
 => CACHED [amf 5/7] RUN mkdir -p amf/                                                                                                                                            0.0s
 => CACHED [amf 6/7] COPY --from=builder /go/src/amf/bin/* ./amf                                                                                                                  0.0s
 => CACHED [amf 7/7] WORKDIR /free5gc/amf                                                                                                                                         0.0s
 => exporting to image                                                                                                                                                            0.0s
 => => exporting layers                                                                                                                                                           0.0s
 => => writing image sha256:9d396c66eb7645f809707ca78ab45d85b87eeac2057dc655d1013307bba2c7d0                                                                                      0.0s
 => => naming to docker.io/library/5gc-amf:0.0.1-dev                                                                                                                              0.0s
rm -rf vendor
```